### PR TITLE
FGM001 changes

### PR DESCRIPTION
**Pull request by saravanakumar-palanichamy**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/utils/BulkScanValidationUtil.java [hunk 1]
```diff
@@ -13,14 +14,19 @@ private BulkScanValidationUtil() {
...
 
-    public static boolean isDateValid(String dateStr, String format) {
+    public static boolean isDateValid(String fieldName, String dateStr, String format) {
         DateFormat sdf = new SimpleDateFormat(format, Locale.UK);
         sdf.setLenient(false);
         try {
-            sdf.parse(dateStr);
+            Date parse = sdf.parse(dateStr);
+            Date current = new Date();
+            if ((fieldName.contains("dateOfBirth") || fieldName.contains("dob")) && parse.after(current)) {
```
Comment (parvezhmcts) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/utils/BulkScanValidationUtil.java:23]: use contants for "dateOfBirth" and "dob"

File: src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java [hunk 2]
```diff
@@ -285,6 +285,123 @@ public static List<OcrDataField> getA60OrC63orA58Data() {
         return fieldList;
     }
 
+    public static List<OcrDataField> getFgm001Data() {
+        List<OcrDataField> fieldList = new ArrayList<>();
+        OcrDataField ocrDataFirstNameField = new OcrDataField();
```
Comment (parvezhmcts) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java:290]: i see same value is used everywhere in the file, so it's better to create constants and use it.

**Review by parvezhmcts — COMMENTED** [review]
generic review comment is replace hard coded string with constants in applicable files.

File: src/main/java/uk/gov/hmcts/reform/bulkscan/utils/BulkScanValidationUtil.java (hunk 1 above)
Comment (saravanakumar-palanichamy) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/utils/BulkScanValidationUtil.java]: moved to constants

File: src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java (hunk 2 above)
Comment (saravanakumar-palanichamy) [reply src/test/java/uk/gov/hmcts/reform/bulkscan/utils/TestDataUtil.java:296]: I moved my changes into comments.

**Review by parvezhmcts — APPROVED** [review]
looks good.

**Pull request merged**